src/SymfonyContainerBridge.php

```diff
@@ -24,7 +24,7 @@
  *
  * @author Matthieu Napoli <matthieu@mnapoli.fr>
  */
-class SymfonyContainerBridge extends SymfonyContainer implements SymfonyContainerInterface, ContainerInterface
+class SymfonyContainerBridge extends SymfonyContainer implements SymfonyContainerInterface
```

Comment:
This removal is because the interface is already implemented in Symfony right?

Is it possible to keep it anyway (for future-proofing)?

Reply:
Yes, I remove it because it already implemented in Symfony from the version 3.3.
But, it's allowed by PHP (no issue on an internal project with PHP 7.1), but some tools (PHPStorm...) display a warning.

We can also restore the original interface (Interop\Container\ContainerInterface instead of the PSR11) to avoid a BC break. (SF implements the PSR11).

Reply:
No it's good to target PSR-11 only. So if I understand correctly can you add:

`implements SymfonyContainerInterface, ContainerInterface`

(`ContainerInterface` would be PSR-11)